sailor/sap_iot/write.py

```diff
@@ -68,14 +70,54 @@ def _upload_data_single_indicator_group(dataset, indicator_set, group_id, templa
         _upload_data_single_equipment(data_subset, equipment.id, tags)
 
 
-def upload_indicator_data(dataset: TimeseriesDataset):
+def _check_indicator_group_is_complete(uploaded_indicators, group_id, template_id):
+    missing = []
+    ig_id = group_id.replace('IG_', '')
```

Comment:
I'd like to not have places where this liot_id vs asset_central_id mapping needs to be done distributed across the code-base.
maybe cleanest would be to also add the `indicator_group_id` to `query_groups` in line 127, and then pass AC ID rather than liot-id to this function?
(And while we're at it, we should probably make the naming between liot-id and ac-id clearer, and not have `group_id` where you don't know which one it is)